src/controllers/rewrite.ts

```diff
@@ -1,14 +1,34 @@
 import { Body, Controller, Get, Post } from 'amala'
-import RewriteBody from '@/validators/RewriteBody'
+// import { ChatGPTAPI } from 'chatgpt' // Use this import in DEV mode to have types
+import Persona from '@/models/Persona'
+import RewriteBody from '@/validators/RewriteBody.js'
+import dynamicImport from '@/helpers/dynamicImport'
+import env from '@/helpers/env.js'
 
 @Controller('/')
 export default class LoginController {
   @Get('/')
   index() {
-    return 'Nothing to see here, move along.'
+    return 'Nothing to see here 👀'
   }
   @Post('/rewrite')
-  rewrite(@Body({ required: true }) { persona, text }: RewriteBody) {
-    return { text, persona }
+  async rewrite(
+    @Body({ required: true }) { persona, text, textLength }: RewriteBody
+  ) {
+    try {
+      const { ChatGPTAPI } = await dynamicImport('chatgpt')
+      const api = new ChatGPTAPI({
+        apiKey: env.OPEN_AI_API_KEY,
+        completionParams: { model: 'gpt-4' }, // or 'gpt-4-0314'
+        maxModelTokens: 8100, // not 32000 because it's not yet available via API
```

Comment:
You pass the same value from the frontend as `textLength`. Why do we need `textLength` then?)